GameData/RealismOverhaul/Localization/en-us-Mfr.cfg

```diff
@@ -206,11 +234,14 @@ Localization
 		#roMfrAirbusDefence = Airbus Defence and Space	//(2017-present) defense division of EADS/Airbus Group
 		#roMfrAirbusGroup = Airbus Group	//(2014-present) holding company
 		#roMfrAirbusIndustrie = Airbus Industrie	//(1970-2001) conglomerated under EADS in 2000, reorganized into Airbus 2001
+		#roMfrArianeGroup = ArianeGroup //(2015-present) Joing Venture between Airbus and Safran, builds Ariane 5 and 6 for ArianeSpace
+		#roMfrArianeSpace = ArianeSpace //(1980-present) Operates the Ariane and Vega line of Rockets
```

Comment:
Unless they actually manufacture hardware and/or are a prime contractor, they shouldn't be here.

Reply:
ArianeGroup does actually manufacture hardware, though I will remove ArianeSpace since that's just marketing